packages/web/src/provider/in-memory-provider/in-memory-provider.ts

```diff
@@ -38,8 +38,7 @@ export class InMemoryProvider implements Provider {
    */
   async putConfiguration(flagConfiguration: FlagConfiguration) {
     try {
-      const flagsChanged = Object.entries(flagConfiguration)
-        .filter(([key, value]) => this._flagConfiguration[key] !== value)
+      const flagsChanged = Object.entries({...flagConfiguration, ...this._flagConfiguration})
```

Comment:
Since you are trying to merge old and new configs, shouldn't the new config come last?
```suggestion
      const flagsChanged = Object.entries({...this._flagConfiguration,...flagConfiguration })
```

Reply:
maybe need a test case for this?

Reply:
reverted